senderle/c18topics
# Add ability to run app using webpack

### Add ability to run app using webpack. 

**Date:** 2/24/17

**Purpose:** This installs and configures a bundler so that we can run the app in a more complex way than just with a simple server. I think this will allow us to modify the CSS more simply, as well as keep us out of trouble as it starts to grow. 

**State of PR:** This is a complete feature.

**Request:** Please QA this feature and check to make sure the code is clean and correct.

**To Test**

1. Run `npm install`. I think this will install all of the necessary packages for you. 
2. run `npm start` and check on localhost:8080 if it is working.
